overlays/holo-nixpkgs/hpos-led-manager/src/main.rs

```diff
@@ -41,21 +44,42 @@ fn main() -> Fallible<()> {
     let state_path = args.flag_state;
     let state_temp_path = state_path.with_extension("tmp");
 
+    let mut counter: u8 = 0;
+    let mut conn_kitsune_proxy = true;
+
     loop {
         let router_gateway_addrs = "router-gateway.holo.host:80".to_socket_addrs();
-        let online = match router_gateway_addrs {
+        let conn_zerotier = match router_gateway_addrs {
             Ok(mut addrs) => match addrs.next() {
                 Some(addr) => TcpStream::connect_timeout(&addr, Duration::new(1, 0)).is_ok(),
                 None => false,
             },
             Err(_) => false,
         };
 
+        if counter % 30 == 0 {
+            conn_kitsune_proxy = match Command::new("proxy-cli")
+                .args(&["--", &args.flag_kitsune])
+                .output()
+            {   
+                Ok(output) => { 
+                    let output_string = String::from_utf8(output.stdout)?;
+                    output_string.contains("tokio_task_count")
+                },
+                Err(_) => false,
+            };
+
+            counter = 0;
+        };
+        
+        counter = counter + 1;
```

Comment:
```suggestion
        if counter == 0 {
            conn_kitsune_proxy = match Command::new("proxy-cli")
                .args(&["--", &args.flag_kitsune])
                .output()
            {   
                Ok(output) => { 
                    let output_string = String::from_utf8(output.stdout)?;
                    output_string.contains("tokio_task_count")
                },
                Err(_) => false,
            };
        };
        
        counter = (counter + 1) % 30;
```

this is how I would do it but your way is fine too